goTezosDelegation.go

```diff
@@ -115,24 +115,34 @@ func (this *GoTezos) GetDelegateRewardsForCycle(delegatePhk string, cycle int) (
 
 //A private function to fill out delegation data like gross rewards and share.
 func (this *GoTezos) getContractRewardsForDelegate(delegatePhk, totalRewards string, cycle int) ([]ContractRewards, error) {
+
 	var contractRewards []ContractRewards
+	
 	delegations, err := this.GetDelegationsForDelegateByCycle(delegatePhk, cycle)
 	if err != nil {
 		return contractRewards, err
 	}
+	
+	bigIntRewards, err := strconv.Atoi(totalRewards)
```

Comment:
moved this here cause we only need to do this calculation once; not for each delegate